src/Compilers/CSharp/Test/Emit/Emit/InAttributeModifierTests.cs

```diff
@@ -4413,10 +4413,12 @@ public class Test
     public ref readonly int M() => throw null;
 }";
 
-            CreateCompilation(user, references: new[] { ref1, ref2 }).VerifyDiagnostics(
-                // (4,12): error CS0518: Predefined type 'System.Runtime.InteropServices.InAttribute' is not defined or imported
-                //     public ref readonly int M() => throw null;
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "ref readonly int").WithArguments("System.Runtime.InteropServices.InAttribute").WithLocation(4, 12));
+            var comp = CreateCompilation(user, references: new[] { ref1, ref2 }).VerifyDiagnostics();
```

Comment:
>CreateCompilation [](start = 23, length = 17)

Perhaps we should also keep the flavor of the test with ambiguity, i.e. by targeting core library without the type.